tests/xtro-sharpie/Makefile

```diff
@@ -225,7 +225,12 @@ $(foreach platform,$(DOTNET_PLATFORMS),$(eval $(call DotNetClassify,$(platform),
 ifdef INCLUDE_IOS
 INCLUDED_PLATFORMS+=iOS
 CLASSIFY+=.stamp-classify-ios
-.stamp-classify-ios: bin/Debug/xtro-sharpie.exe $(XIOS_PCH) $(XIOS) $(XIOS_GL)
+STAMP_CLASSIFY_IOS_DEPS := bin/Debug/xtro-sharpie.exe $(XIOS_PCH) 
+ifndef TESTS_USE_SYSTEM
+STAMP_CLASSIFY_IOS_DEPS += $(XIOS)
+STAMP_CLASSIFY_IOS_DEPS += $(XIOS_GL)
+endif
```

Comment:
The reason you had to do this change was because the dll doesn't exist... but that doesn't make things work, it just makes xtro not run correctly.